src/viztracer/modules/vcompressor/vc_dump.c

```diff
@@ -258,3 +261,123 @@ load_events_from_file(FILE* fptr)
     return parsed_events;
 
 }
+
+
+int dump_file_info(PyObject* file_info, FILE* fptr){
+
+    // dump json string
+    uint64_t content_length = 0;
+    uint64_t compression_length = 0;
+
+    PyObject* json_ret = NULL;
+    PyObject* zlib_ret = NULL;
+    PyObject* bytes_data = NULL;
+    PyObject* json_module_name = PyUnicode_FromString("json");
+    PyObject* json_module = PyImport_Import(json_module_name);
+    PyObject* dumps_func = PyObject_GetAttrString(json_module, "dumps");
+    PyObject* json_args = PyTuple_New(1);
+    PyObject* zlib_module_name = PyUnicode_FromString("zlib");
+    PyObject* zlib_module = PyImport_Import(zlib_module_name);
+    PyObject* compress_func = PyObject_GetAttrString(zlib_module, "compress");
+    PyObject* zlib_args = PyTuple_New(1);
+    
+    // json dumps file_info
+    PyTuple_SetItem(json_args, 0, file_info);
+    Py_INCREF(file_info); 
```

Comment:
Thanks a lot for reviewing, but the file_info here points the file_info in json object that we compress. If `Py_INCREF(file_info)` is removed, the file_info in json object will be released after compress because `json_args` is released.